src/DDTrace/Scope.php

```diff
@@ -5,7 +5,7 @@
 use OpenTracing\Scope as OpenTracingScope;
 use OpenTracing\Span as OpenTracingSpan;
 
-final class Scope implements OpenTracingScope
+class Scope implements OpenTracingScope
```

Comment:
What's this change for?

Reply:
Sorry, I was investigating some errors I got when trying to run PHPUnit testcases for the first time. And its a leftover of this investigating. I'll revert it.
 Thanks for finding that!